cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/dumb-init /bin/sh
+#  ----------------------------------------------------------------------------------
+#  Copyright (c) 2021 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#  ----------------------------------------------------------------------------------
+
+# This is customized entrypoint script for Consul.
+# In particular, it waits for Vault to be ready to roll
+
+set -e
+
+# function to check on Vault for readiness
+vault_ready()
+{
+  vault_host=$1
+  vault_port=$2
+  resp_code=$(curl --write-out '%{http_code}' --silent --output /dev/null "${vault_host}":"$vault_port"/v1/sys/health)
+  if [ "$resp_code" -eq 200 ] ; then
+    echo 1
+  else
+    echo 0
+  fi
+}
+
+# env settings
+. /edgex-init/.env-consul
+
+echo "Script for waiting security bootstrapping installation"
+
+echo "$(date) Consul waits on Vault to be initialized"
+vault_inited=$(vault_ready "${EDGEX_VAULT_HOST}" "$EDGEX_VAULT_PORT")
+until [ "$vault_inited" -eq 1 ]; do
+    echo "$(date) waiting for Vault to be initialized";
+    sleep 1;
+    vault_inited=$(vault_ready "${EDGEX_VAULT_HOST}" "$EDGEX_VAULT_PORT")
+done
+
+echo "$(date) Starting edgex-consul..."
+exec /usr/local/bin/edgex-consul-entrypoint.sh agent -ui -bootstrap -server -client 0.0.0.0 &
+
+# wait for the consul port
+echo "$(date) Executing dockerize on Consul with waiting on its own port tcp://${CONSUL_HOST}:$CONSUL_PORT"
+/edgex-init/dockerize -wait tcp://"${CONSUL_HOST}":"$CONSUL_PORT" -timeout 30s
```

Comment:
This script will be executing from the Consul container. Will `/edgex-init` be the volume mount point?

Reply:
yes